.github/workflows/provenance.yaml

```diff
@@ -18,6 +21,7 @@ jobs:
       OAK_FUNCTIONS_BASE_PATH: './target/x86_64-unknown-linux-musl/release/oak_functions_loader_base'
       OAK_FUNCTIONS_UNSAFE_PATH: './target/x86_64-unknown-linux-musl/release/oak_functions_loader_unsafe'
       OAK_BAREMETAL_CROSVM_PATH: './experimental/oak_baremetal_app_crosvm/target/x86_64-unknown-none/release/oak_baremetal_app_crosvm'
+      OAK_FUNCTIONS_RELEASE_PATH: './target/x86_64-unknown-linux-musl/release'
```

Comment:
We use the we could use the `OAK_FUNCTIONS_RELEASE_PATH` in `OAK_FUNCTIONS_BASE_PATH` and `OAK_FUNCTIONS_UNSAFE_PATH`.

Reply:
Exactly. I thought I'd fix this after I am done with the experiment :)